OWNERS

```diff
@@ -1,10 +1,16 @@
 # Each list is sorted alphabetically, additions should maintain that order
```

Comment:
can we please move this change to servicemesh directory?
providing owner access to base layer would not be possible.

move the same change under the servicemesh folder.

Reply:
Sure, that's done now (70576dd) in the following locations:

  * `odh-common/overlays/service-mesh/OWNERS`
  * `odh-dashboard/overlays/service-mesh/OWNERS`
  * `odh-notebook-controller/overlays/service-mesh/OWNERS`        
  * `odh-project-controller/OWNERS`
  *  `service-mesh/OWNERS`

Reply:
thanks :+1: